src/adk/api/remote_api.py

```diff
@@ -789,23 +824,24 @@ def __translate_asset(asset_to_create: AssetType, experiment_url: str) -> AssetT
         asset_to_create["experiment"] = experiment_url
         return asset_to_create
 
-    def __create_round_set(self, asset_url: str, number_of_rounds: int) -> RoundSetType:
+    def __create_round_set(self, asset_url: str, backend_type_url: str, number_of_rounds: int) -> RoundSetType:
```

Comment:
why are you using the backendtype URL (as opposed to the name?)

Reply:
API-router expects this because the serializers on experiment/asset/roundset are subclasses of HyperlinkedModelSerializer in the rest framework. We use these so we can use the url's instead of the key values (as the normal ModelSerializer expects). Using the url's directly makes it easier for front ends as QNE-ADK and the website to retrieve resources.